client/api/dbs/dbs.py

```diff
@@ -0,0 +1,198 @@
+    id = db.Column(db.Integer, primary_key=True)
+    user_id = db.Column(db.Integer, db.ForeignKey("Users.id"), nullable=False)
+    email = db.Column(db.String(100), nullable=False)
+    first_name = db.Column(db.String(50))
+    last_name = db.Column(db.String(50))
+    profile_pic = db.Column(db.BLOB)
+
+    def updateUserProfile(user_id, email, first_name, last_name, profile_pic):
+        profile = UserProfile.query.filter_by(user_id=user_id).first()
+        if not profile:
+            return "Profile Not Found"
+        else:
+            if email != "":
+                profile.email = email
+            if first_name != "":
+                profile.first_name = first_name
+            if last_name != "":
+                profile.last_name = last_name
+            if profile_pic is not None:
+                profile.profile_pic = profile_pic
+            db.session.commit()
+            return "Profile Successfully Updated"
+
+    def getUserProfile(user_id):
+        userProfile = UserProfile.query.filter_by(user_id=user_id).first()
+        if not userProfile:
+            return "Profile Not Found"
+        else:
+            return userProfile
+
+
+class MediaTypes(db.Model):
+    id = db.Column(db.Integer, primary_key=True)
+    name = db.Column(db.String(50))
+
+    def getAllMediaTypes():
+        mediaTypes = MediaTypes.query.all()
+        return mediaTypes
+
+    def getMediaType(id):
+        mediaType = MediaTypes.query.filter_by(id=id).first()
+        if not mediaType:
+            return "Media Type Not Found"
+        else:
+            return mediaType
+
+
+class Lists(db.Model):
+    id = db.Column(db.Integer, primary_key=True)
+    user_id = db.Column(db.Integer, db.ForeignKey("Users.id"), nullable=False)
+    list_name = db.Column(db.String(100), nullable=False)
+    media_type_id = db.Column(
+        db.Integer, db.ForeignKey("MediaTypes.id"), nullable=False
+    )
+    description = db.Column(db.String(250))
+
+    def addList(user_id, list_name, media_type_id, description):
+        new_list = Lists(
+            user_id=user_id,
+            list_name=list_name,
+            media_type_id=media_type_id,
+            description=description,
+        )
+        db.session.add(new_list)
+        db.session.commit()
+
+    def editList(id, list_name, description):
+        list = Lists.query.filter_by(id=id).first()
+        if not list:
+            return "List Not Found"
+        else:
+            if list_name != "":
+                list.list_name = list_name
+            if description != "":
+                list.description = description
+            db.session.commit()
+            return "List Updated"
+
+    def deleteList(id):
+        Lists.query.filter_by(id=id).delete()
+        db.session.commit()
+
+
+class ListItems(db.Model):
+    id = db.Column(db.Integer, primary_key=True)
+    list_id = db.Column(db.Integer, db.ForeignKey("Lists.id"), nullable=False)
+    media_id = db.Column(
```

Comment:
Is this supposed to be an element we want from my APIs or is this meant to be from the MediaTypes database?